MSVC bit can exceed size for `_Bool`

MSVC seems ok with this. I assume it's because bools are stored as char.
```c
typedef struct {
    _Bool v:8;
} S;

_Static_assert(sizeof(S) == 1, "");
_Static_assert(_Alignof(S) == 1, "");
```
arocc gives the error
```
test/records/0036_test.c:9:11: error: width of
bit-field exceeds width of its type
    _Bool v:8;
          ^
```

--- a/src/Type.zig
+++ b/src/Type.zig
@@ -817,7 +817,7 @@ pub fn sizeof(ty: Type, comp: *const Compilation) ?u64 {
 
 pub fn bitSizeof(ty: Type, comp: *const Compilation) ?u64 {
     return switch (ty.specifier) {
-        .bool => 1,
+        .bool => if (comp.langopts.emulate == .msvc) @as(u64, 8) else 1,
         .typeof_type, .decayed_typeof_type => ty.data.sub_type.bitSizeof(comp),
         .typeof_expr, .decayed_typeof_expr => ty.data.expr.ty.bitSizeof(comp),
         .attributed => ty.data.attributed.base.bitSizeof(comp),

--- a/test/cases/msvc boolean bitfield.c
+++ b/test/cases/msvc boolean bitfield.c
@@ -1,0 +1,12 @@
+//aro-args --target=x86_64-windows-msvc
+
+typedef struct {
+    _Bool a: 4;
+    _Bool : 2;
+    _Bool c: 2;
+} S;
+
+// _Static_assert(sizeof(S) == 1, "");
+_Static_assert(_Alignof(S) == 1, "");
+
+#define TESTS_SKIPPED 1

